content/en/docs/concepts/storage/volumes.md

```diff
@@ -338,71 +294,107 @@ spec:
-and then serve it in parallel from as many Pods as you need. Unfortunately,
-PDs can only be mounted by a single consumer in read-write mode. Simultaneous
-writers are not allowed.
+### hostPath {#hostpath}
+
+A `hostPath` volume mounts a file or directory from the host node's filesystem
+into your pod. This volume type improperly breaks the boundary betwen host and container. However, it may be necessary in certain situations, such as testing or running legacy applications. 
+
+For example, some uses for a `hostPath` are:
 
-Using a GCE persistent disk with a Pod controlled by a ReplicaSet will fail unless
-the PD is read-only or the replica count is 0 or 1.
+* running a container that needs access to Docker internals; use a `hostPath`
+  of `/var/lib/docker`
+* running cAdvisor in a container; use a `hostPath` of `/sys`
+* allowing a pod to specify whether a given `hostPath` should exist prior to the
+  pod running, whether it should be created, and what it should exist as
 
-#### Creating a GCE persistent disk {#gce-create-persistent-disk}
+In addition to the required `path` property, you can optionally specify a `type` for a `hostPath` volume.
 
-Before you can use a GCE persistent disk with a Pod, you need to create it.
+The supported values for field `type` are:
 
-```shell
-gcloud compute disks create --size=500GB --zone=us-central1-a my-data-disk
-```
+| Value | Behavior |
+|:------|:---------|
+| | Empty string (default) is for backward compatibility, which means that no checks will be performed before mounting the hostPath volume. |
+| `DirectoryOrCreate` | If nothing exists at the given path, an empty directory is created there as needed with permission set to 0755, having the same group and ownership with Kubelet. |
+| `Directory` | A directory must exist at the given path |
+| `FileOrCreate` | If nothing exists at the given path, an empty file is be created there as needed with permission set to 0644, having the same group and ownership with Kubelet. |
+| `File` | A file must exist at the given path |
+| `Socket` | A UNIX socket must exist at the given path |
+| `CharDevice` | A character device must exist at the given path |
+| `BlockDevice` | A block device must exist at the given path |
 
-#### GCE persistent disk configuration example
+Watch out when using this type of volume, because:
+
+* Pods with identical configuration (such as those created from a PodTemplate) may
+  behave differently on different nodes due to different files on the nodes
+* The files or directories created on the underlying hosts are only writable by root. You
+  either need to run your process as root in a
+  [privileged container](/docs/tasks/configure-pod-container/security-context/) or modify the file
+  permissions on the host to be able to write to a `hostPath` volume
+
+#### Resources
+
+The storage media (such as Disk or SSD) of an `emptyDir` volume is determined by the medium of the filesystem holding the kubelet root dir (typically
```

Comment:
Move the references to `emptyDir` to the emptydir section?

Reply:
done, thank you for noticing this